backend/de.metas.payment.sepa/base/src/main/java/de/metas/payment/sepa/api/impl/CreateSEPAExportFromPaySelectionCommand.java

```diff
@@ -104,7 +106,16 @@ private I_SEPA_Export_Line createExportLine(@NonNull final I_C_PaySelectionLine
 		exportLine.setC_BPartner_ID(line.getC_BPartner_ID());
 		exportLine.setDescription(sourceInvoice.getDescription());
 
-		exportLine.setIBAN(toNullOrRemoveSpaces(bpBankAccount.getIBAN()));
+		final String IBAN;
+		if (toNullOrRemoveSpaces(bpBankAccount.getIBAN()) != null)
+		{
+			IBAN = toNullOrRemoveSpaces(bpBankAccount.getIBAN());
+		}
+		else
+		{
+			IBAN = toNullOrRemoveSpaces(bpBankAccount.getQR_IBAN());
+		}
```

Comment:
```suggestion
		final String IBAN = toNullOrRemoveSpaces(coalesce(bpBankAccount.getIBAN(), bpBankAccount.getQR_IBAN()));
```

With a static import for `de.metas.common.util.CoalesceUtil.coalesce`